tests/php/ORM/HierarchyTest.php

```diff
@@ -682,4 +682,14 @@ public function testDefaultParent(string $class, ?string $defaultParentConfig, ?
 
         $this->assertSame($expected, $obj->defaultParent());
     }
+
+    /**
+     * Tests that HTML added by an extension is not escaped
+     */
+    public function testGetTreeTitleExtension()
+    {
+        HierarchyTest\TestObject::add_extension(HierarchyTest\TestTreeTitleExtension::class);
```

Comment:
The rest of the file used the  HierarchyTest\{class} style instead of imports, so just being consistent